library/src/main/java/com/mapzen/android/MapzenMap.java

```diff
@@ -280,16 +309,30 @@ public LngLat coordinatesAtScreenPosition(double screenX, double screenY) {
     /**
      * Set tap responder for tap gestures on map.
      */
-    public void setTapResponder(final TouchInput.TapResponder tapResponder) {
-        this.tapResponder = tapResponder;
-        mapController.setTapResponder(this.tapResponder);
+    public void addTapResponder(final TouchInput.TapResponder tapResponder) {
```

Comment:
Changed.

I had changed this so that setting a `FeaturePickListener` could encapsulate calling `pickFeature` but realize I can just set an internal `TapResponder` and have it manage a single external responder plus the call to `pickFeature`. This will keep our api on par with GoogleMap's!